src/API/Poll.vala

```diff
@@ -0,0 +1,75 @@
+using Gee;
+public class Tootle.API.Poll {
+    public string id { get; set; }
+    public string expires_at{ get; set; }
+    public bool expired { get; set; }
+    public bool multiple { get; set; }
+    public int64 votes_count { get; set; }
+    public int64 voters_count { get; set; }
+    public bool voted { get; set; }
+    public int64[] own_votes { get; set; }
```

Comment:
This makes me wish there was a way to (de)serialize Vala arrays automatically, just like `Entity.vala` does with `Gee.ArrayList<Object>`'

Maybe we could do something here so that we can omit the boilerplate code for parsing.